lib/new_relic/control/instrumentation.rb

```diff
@@ -98,6 +99,21 @@ def ruby_22_deprecation
       ::NewRelic::Agent.record_metric("Supportability/Deprecated/Ruby22", 1)
     end
 
+    def ruby_23_deprecation
```

Comment:
I'd prefer to see a combined "support for Ruby versions below 2.4" message instead of these separate 2.2 and 2.3 ones.

As currently implemented, if I was on 2.2 and saw the 2.2 specific message, I might think upgrading to 2.3 would get me into the clear. But then I'd run into the 2.3 specific message as well.

Reply:
Great point! Resolved in: 26be8d344d1bb3dc28fdfbc8400ad3eff628d9da